mesa/model.py

```diff
@@ -90,14 +117,23 @@ def load_totals(self):
             key = model.areas[i]
             self.area_population[key] = population[i] / population_sum * self.total_citizens
             self.area_names[key] = names[i]
+        self.population_details_db = self.read_csv_city("neukolln-by-citizenship-migrantbackground-gender-religion-age")
+        for a in self.areas:
+            details = []
+            for index, row in self.population_details_db.iterrows():
```

Comment:
In generale `pd.iterrows()` è rinomata per essere estremamente lenta, meglio vettorizzare. Ma credo che questa procedura viene attivata solo una volta e il dataframe è anche piccolo, quindi nessun problema qui.